lib/types/CustomerB2BUser.ts

```diff
@@ -0,0 +1,31 @@
+import type { Maybe, ProductOptionSelectionInput, CrWishlist } from '@/lib/gql/types'
+export interface B2BUserInput {
+  firstName: string
+  lastName: string
+  emailAddress: string
+  userName: string
+  localeCode?: string
+}
+
+// export interface DeleteWishlistItemInput {
+//     product: WishlistProductInput
+//     currentWishlist?: Maybe<CrWishlist>
+// }
+// export interface WishlistParams extends WishlistProductInput {
+//     currentWishlist?: Maybe<CrWishlist>
+// }
+// export interface WishlistItemInWishlistParams {
+//     productCode: string
+//     variationProductCode?: string
+//     userWishlist?: Maybe<CrWishlist>
+// }
+
+export interface CustomerB2BUserParams {
+  removeCustomerB2bAccountUser?: boolean
+  delay?: number
+}
+
+export interface CustomerB2BUserRole {
+  roleName: string
+  roleId: number
+}
```

Comment:
Remove commented code it its not required

Reply:
Done